raycicmd/bk_pipeline.go

```diff
@@ -143,11 +143,12 @@ func makeRayDockerPlugin(
 }
 
 // makeAutomaticRetryConfig creates the retry configuration for rayci pipelines.
-// The retry configuration is to retry once for any unknown exit status, and
-// to retry 3 times for known exit statuses.
+// The retry configuration is to retry once for any unknown exit status or test failures,
+// and to retry 3 times for known exit statuses.
```

Comment:
as there are two cases(?) could you add inline comment (around 42) which is which?

Reply:
added comments